packages/frontend-core/src/components/grid/cells/RelationshipCell.svelte

```diff
@@ -322,6 +357,21 @@
   </GridPopover>
 {/if}
 
+{#if relationshipFields}
```

Comment:
NAB/performance optimisation: Does this work the same if we instead do `{#if showRelationshipFields}...`? Just think that otherwise we actually end up and unmounting this grid popover rapidly when moving the mouse over the grid, even if the cells aren't focused (since we still have the mouseover handler). Updating the if statement to use `showRelationshipFields` would ensure it only gets mounted when we intend to show it. I thiynk we can just leave the `open` prop out then, since it defaults to true.